scripts/update

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${GEOSITE_DEBUG}" ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n "Usage: $(basename "$0")
+    Creates app container if nonexistant and attempts to install dependencies
+    "
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]
+then
+    if [ "${1:-}" = "--help" ]
+    then
+        usage
+    else
+        if docker-compose --version; then
+            docker-compose build
```

Comment:
If we are standardizing on a single set of scripts, I think this can just do a `pip install` from the requirements.txt file